wweval/R/eval_fit.R

```diff
@@ -94,46 +76,73 @@ eval_fit_ww <- function(config_index,
     calibration_time = eval_config$calibration_time,
     last_hosp_data_date = last_hosp_data_date,
     ww_data_mapping = eval_config$ww_data_mapping
-  )
+  ) |>
+    dplyr::mutate(
+      "location" = !!location,
+      forecast_date = forecast_date
+    )
```

Comment:
Similar to above. Incorporate into the `get_input_ww_data` function in a separate PR.

Reply:
I did this in this PR!